.github/workflows/python-package.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
```

Comment:
What is the subtle difference between `CI.yml` and `python-package.yml`? Why do we need both? Is `python-package.yml` even executed?

Reply:
No, this was added in error!